src/components/forms/Forms.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import CreateForm from "./CreateForm";
+import AddTransactionsForm from "./AddTransactionsForm";
+
+const isActivityDataEntered = () => {
+  const getActivityData = JSON.parse(localStorage.getItem("activityDetails"));
+  if (getActivityData) {
+    return true;
+  }
```

Comment:
```suggestion
  if (getActivityData) return true;
```

curly braces are not necessary for one liners, feel free to use it if you like it